.stdci.yml

```diff
@@ -5,4 +5,4 @@ Architectures:
       runtime-requirements:
         host-distro: same
 script:
-    from-file: automation/run-integration-tests.sh
+    from-file: env TEST_TYPE=integ automation/run-tests.sh
```

Comment:
Is this really working? I thought it only accepts files as values.

Reply:
Changed to use argument:
```
from-file: automation/run-tests.sh --test-type integ
```